src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/page/ViewBacklogPage.kt

```diff
@@ -38,4 +40,8 @@ class ViewBacklogPage(
             textToBePresentInElementLocated(By.cssSelector(".ghx-backlog-container .ghx-name"), expectedSprintName)
         )
     }
+
+    fun closeAuiFlags() {
+        popUps.waitUntilAuiFlagsAreGone()
```

Comment:
```suggestion
    fun closePopups() {
        popUps.waitUntilAuiFlagsAreGone()
```
This way the caller doesn't need to know that backlog page suffers from AUI flags. If it turns out that more popups are getting in the way (now or in the future Jira versions), then this method can update its implementation without forcing consumers to update their knowledge and calls.

Reply:
I kinda disagree on the achitectural level with details of such design, however I doubt that I have enough skill to convince you in a single comment. We could discuss advantages of some other concept that I'd like us to follow on other occasion if there will be time. Keywords would be: "Hiding the behavior".

I don't see the suggestion making the current situation worse, so I'm completely fine with it.